Angular/Getting started with Angular.md

```diff
@@ -0,0 +1,87 @@
+
+After you finish the tutorial you will no doubt be left with questions. These guides,alongside your mentor, can help you find those answers.
+
+Here are some of the most important guides:
+
+- [Dependency Injection in Angular](https://angular.io/guide/dependency-injection)
+- [Routing & Navigation](https://angular.io/guide/router)
+- [Animations](https://angular.io/guide/animations)
+- [Testing](https://angular.io/guide/testing)
+- [Observables](https://angular.io/guide/observables)
+- [The RxJS library](https://angular.io/guide/rx-library)
+
+Most of these are quite large documents. Nobody expects you to read through all the contents of all these documents. But you should know they are there so you can come back to them when you are unsure of how some things work. Even the most experienced developers use the documentation pages on a regular basis to remind themselves of some parts of the framework which they might not have used as extensively in a while so they need a refreshed.
+
+### 1.3. Reference pages
+
+If you have some specific questions
+
+## 2. Infinum [Angular guidelines and best practices](https://handbook.infinum.co/books/frontend/javascript/angular) Handbook
+
+The official docs are a great source of reference information, guides and tutorials. After you finish all the official tutorials, taking a look at our very own [Angular Handbook](https://handbook.infinum.co/books/frontend/javascript/angular). At first we recommend you skim-read it on a high-level, just to get to know which topics are covered. Of course feel free to read it completely if you have the time.
+
+The Handbook covers some of the best practices and formatting preferences to which we try and adhere when possible. This includes:
+
+- Scaffolding
+- File organization
+- Tooling
+- Editor and editor extensions recommendations
+- RxJS basics
+- Ways to conceptually organize components
+- Handling multiple deployment environments
+- Formatting, naming and best practices in Angular templates and TypeScript
+- Dependency injection links
+- Angular universal links
+- Two-way binding
+- Working with forms
+- Testing
+
+One of the topics which is usually not covered into detail in most online tutorials is testing. A large portion of the Handbook is about [testing](https://handbook.infinum.co/books/frontend/javascript/angular#testing) and this should come in very handy once you start working on an actual project where it might be required to write tests.
+
+## 3. Get to know RxJS
+
+There are many concepts which Angular shares with React, Vue and other component-based JavaScript libraries/frameworks. However, when it comes to Angular, there is one big player in the game who is usually not present when using other frameworks, and that is RxJS.
+
+Writing good quality Angular applications both in terms of performance and code readability and reusability often required having a good grasp of how RxJS works and how it can be utilized to create reactive data streams.
```

Comment:
```suggestion
Writing good quality Angular applications both in terms of performance and code readability and reusability often requires having a good grasp of how RxJS works and how it can be utilized to create reactive data streams.
```